sigs.yml

```
@@ -0,0 +1,502 @@
+    chat:
+      - type: slack
+        name: '#otel-comms'
+        id: C02UN96HZH6
+    invites: calendar-comms
+    tcSponsors: []
+    gcLiaison:
+    - name: 'Severin Neumann'
+      github: svrnm
+  - name: End-User WG
+    meeting: Every other week on Thursday at 10:00 PT
+    notes:
+      type: gDoc
+      value: 1e-UNZA3Tuno9b53RQbe--whUcO0VIXF3P81oXsrBK6g
+    chat:
+      - type: slack
+        name: '#otel-user-research'
+        id: C01RT3MSWGZ
+    invites: calendar-user-research
+    tcSponsors: []
+    gcLiaison:
+    - name: Daniel Gomez Blanco
+      github: danielgblanco
+  - name: eBPF
+    meeting: Every week on Tuesday at 09:00 PT
+    notes:
+      type: gDoc
+      value: 13GK915hdDQ9sUYzUIWi4pOfJK68EE935ugutUgL3yOw
+    chat:
+      - type: slack
+        name: '#otel-ebpf'
+        id: C02AB15583A
+    invites: calendar-ebpf
+    tcSponsors: []
+    gcLiaison: []
+  - name: Agent Management WG
+    meeting: Every other week on Tuesday at 11:00 PT
+    notes:
+      type: gDoc
+      value: 19WA5-ex8rNFIBIyVb5VqMXfWNmUQwppGhN8zBeNG0f4
+    chat:
+      - type: slack
+        name: '#otel-agentmanwg'
+        id: C02J58HR58R
+    invites: calendar-agent-mgmt
+    tcSponsors: []
+    gcLiaison: []
+  - name: Prometheus Interoperability WG
+    meeting: Every other week on Wednesday at 08:00 PT
+    notes:
+      type: gDoc
+      value: 19bnXziPn2MZ9wO6684UoI4D-LCjGL5bTJkGhux29bx8
+    chat:
+      - type: slack
+        name: '#otel-prometheus-wg'
+        id: C01LSCJBXDZ
+    invites: calendar-prometheus
+    tcSponsors: []
+    gcLiaison: []
+  - name: Client Instrumentation
+    meeting: Every Tuesday at 9:00 AM PT
+    notes:
+      type: gDoc
+      value: 16Vsdh-DM72AfMg_FIt9yT9ExEWF4A_vRbQ3jRNBe09w
+    chat:
+      - type: slack
+        name: '#otel-client-side-telemetry'
+        id: C0239SYARD2
+    invites: calendar-client-side
+    tcSponsors: []
+    gcLiaison: []
+  - name: Kubernetes Operator
+    meeting: Every other week on Thursday at 09:00 PT
+    notes:
+      type: gDoc
+      value: 1Unbs2qp_j5kp8FfL_lRH-ld7i5EOQpsq0I4djkOOSL4
+    chat:
+      - type: slack
+        name: '#otel-operator'
+        id: C033BJ8BASU
+    invites: calendar-k8s-operator
+    tcSponsors: []
+    gcLiaison: []
+  - name: Community Demo Application
+    meeting: Every Wednesday at 8:00 PT
+    notes:
+      type: gDoc
+      value: 16f-JOjKzLgWxULRxY8TmpM_FjlI1sthvKurnqFz9x98
+    chat:
+      - type: slack
+        name: '#otel-community-demo'
+        id: C03B4CWV4DA
+    invites: calendar-demo-app
+    tcSponsors: []
+    gcLiaison:
+    - name: 'Austin Parker'
+      github: austinlparker
+  - name: Functions as a Service (FAAS)
+    meeting: Every other Tuesday at 12:00 PM PT
+    notes:
+      type: gDoc
+      value: 187XYoQcXQ9JxS_5v2wvZ0NEysaJ02xoOYNXj08pT0zc
+    chat:
+      - type: slack
+        name: '#otel-faas'
+        id: C04HVBETC9Z
+    invites: calendar-faas
+    tcSponsors: []
+    gcLiaison: []
```

```suggestion
    gcLiaison: []
```
@open-telemetry/governance-committee please assign yourself for being the liaison person for this SIG